client/components/me-alerts/me-alerts.controller.js

```diff
@@ -0,0 +1,24 @@
+class MeAlertsCtrl {
```

Comment:
Yap, I had in mind to keep the current structure in the way to respect consistency between managers.
Following telecom manager : https://github.com/ovh-ux/ovh-manager-telecom/tree/master/client/components/me-alerts